### ISSUE TITLE
Change notices when pulling/building docker containers

### ISSUE THREAD

**Issue by rcannood** [viash-io/viash#94]
Our current output messages might scare off first time users:

```
$ wget -qO- get.viash.io | bash
[notice] Running 'docker pull viash_viash_install:0.1'
Error response from daemon: pull access denied for viash_viash_install, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
[notice] Running 'docker build -t viash_viash_install:0.1 /tmp/viashsetupdocker-viash_install-1YUosC'
```

it might be helpful to change the notice to catch that and report something like "checking if docker container is available at XXX" and then "Container not found, building from scratch".

Thanks @dburkhardt for the suggestion!

**Comment by BlackDragonBE** [comment]
@rcannood 
I'm all for user friendliness!

Along the same vein of thought, maybe it might be sensible to check if docker is installed at the start of the script? Currently it throws a bunch of `bash: line xxx: docker: command not found` errors if docker isn't installed.   
The script could notify the user with a friendly message like "Docker wasn't found on your system. Please see this page for instructions: https://docs.docker.com/get-docker" for example.

**Comment by rcannood** [comment]
Should be implemented in Viash 0.5.6!

@BlackDragonBE: For unrelated issues or semi-related issues, please create a separate issue next time ;)
